src/blurb/blurb.py

```diff
@@ -818,6 +827,8 @@ def help(subcommand=None):
 
 # Make "blurb --help" work.
 subcommands["--help"] = help
+subcommands["--version"] = version
+subcommands["-v"] = version
```

Comment:
This is perhaps bikeshedding but I commonly see `-v` for verbose mode, and `-V` for version (following `python -V`). Certainly an improvement regardless.
```suggestion
subcommands["-V"] = version
```

Reply:
Yep, good idea!